doc/ApiDocGeneration/Generate-Api-Docs.ps1

```diff
@@ -163,9 +166,11 @@ Write-Verbose "Make changes on relative path on page index.html."
 $baseUrl = $destFolder + "index.html"
 $content = Get-Content -Path $baseUrl -Raw
 $hrefRegex = "[""']\.\.\/([^""']*)[""']"
-$tocRegex = "(./)?toc.html"
+$innerTocRegex = "[""']toc.html[^""']*[""']"
+$outerTocRegex = "[""']\.\.\/toc.html[^""']*[""']"
```

Comment:
Will this ever exist after you replace the relative paths with the first regex?

Reply:
I have added the root toc.yml back above. I found this is used for nav bar which direct to homepage when click. If we use the inner toc in /api, we will have a mess on nav bar.

Reply:
I understand why you want to do this but I'm not sure it actually works. Your first regex replaces `../` and this regex is looking for `../toc.html` but after the first regex replacement you will never have `../` anywhere.

Reply:
That's a valid concern.  I will adjust the replacement order.